Docs: Fix minor mistakes and rephrase "Manage your alert notifications" entry (#68851)
grafana/grafana: docs/sources/alerting/manage-notifications/_index.md

Choosing how, when, and where to send your alert notifications is an important part of setting up your alerting system. These decisions will have a direct impact on your ability to resolve issues quickly and not miss anything important.

> As a first step, define your contact points; where to send your alert notifications to. A contact point is a set of one or more [integrations]({{< relref "./manage-contact-points/configure-integrations/" >}}) that will be used together to deliver notifications. Add notification templates to contact points for reuse and consistent messaging in your notifications.

Reviewer: ```suggestion
As a first step, define your contact points; where to send your alert notifications to. A contact point is a set of one or more [integrations]({{< relref "./manage-contact-points/configure-integrations/" >}}) that are used to deliver notifications. Add notification templates to contact points for reuse and consistent messaging in your notifications.
```
